app/views/layouts/application.html.erb

```diff
@@ -4,41 +4,45 @@
     <title>ObituaryHub</title>
     <%= csrf_meta_tags %>
 
-    <%= stylesheet_link_tag    'application', media: 'all' %>
+    <%= stylesheet_link_tag 'application', media: 'all' %>
     <%= javascript_include_tag 'application' %>
   </head>
 
   <body>
-    <nav1 class="navbar navbar-expand-lg navbar-light style="background-color: #e3f2fd;">
-      <h1><a class="navbar-brand" href="/">Obituary Hub</a></h1>
-      <nav2 class="navbar-right navbar-light bg-light">
-        <form class="form-inline">
-          <% if current_user %>
-            <button class="btn btn-sm btn-outline-secondary" type="button">Logout</button>
+  <nav1 class="navbar navbar-expand-lg navbar-light style="background-color: #e3f2fd;">
+
+     <h1><a class="navbar-brand" href="/">Obituary Hub</a></h1>
+    <nav class="navbar-right navbar-light bg-light">
+      <form class="form-inline">
+        <% if current_user %>
+          <%= link_to "Logout", "/logout", method: :delete, class: "btn btn-sm btn-outline-secondary" %>
             <a href="/obituaries/new"><button class="btn btn-sm btn-outline-secondary" type="button">Add Obituary</button></a>
-          <% else %>
-            <button class="btn btn-sm btn-outline-secondary" type="button">Login</button>
-            <button class="btn btn-sm btn-outline-secondary" type="button">Register</button>
-          <% end %>
-        </form>
-      </nav2>
-    </nav1>
-    <nav3 class="navbar navbar-expand-lg navbar-light bg-light style="background-color: #e3f2fd;">
-      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarText" aria-controls="navbarText" aria-expanded="false" aria-label="Toggle navigation">
-        <span class="navbar-toggler-icon"></span>
-      </button>
-      <div class="collapse navbar-collapse" id="navbarText">
-        <ul class="navbar-nav mr-auto">
-          <li class="nav-item active">
-            <a class="nav-link" href="/obituaries">All Obituaries <span class="sr-only">(current)</span></a>
-          </li>
-          <li class="nav-item">
-            <a class="nav-link" href="/obituaries/covid-19">COVID-19 Obituaries</a>
-          </li>
-          <li class="nav-item">
-            <a class="nav-link" href="/obituaries/recent">Recently Published</a>
-          </li>
-        </ul>
+        <% else %>
+          <a href="/login"<button class="btn btn-sm btn-outline-secondary" type="button">Login</button></a>
+          <a href="/users/new"<button class="btn btn-sm btn-outline-secondary" type="button">Register</button></a>
+        <% end %>
+      </form>
+    </nav>
+  </nav>
```

Comment:
These `nav`s had numbers for testing purposes. I'm confused why the Travis CI build worked because those tests should have been failing.

Reply:
Not sure on that one. It required nav1 to work. I think it is smart enough to just look at the start nav tag. I corrected them regardless and just push it up again